profiles/sssd/system-auth

```diff
@@ -12,33 +12,41 @@ auth        [default=1 ignore=ignore success=ok]         pam_localuser.so
+-auth       sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
 
 account     required                                     pam_access.so                                          {include if "with-pamaccess"}
 account     required                                     pam_faillock.so                                        {include if "with-faillock"}
+-account    sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
 account     required                                     pam_unix.so
 account     sufficient                                   pam_localuser.so                                       {exclude if "with-files-access-provider"}
 account     sufficient                                   pam_usertype.so issystem
 account     [default=bad success=ok user_unknown=ignore] pam_sss.so
 account     required                                     pam_permit.so
+-account    sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
 
+-password    sufficient                                  pam_systemd_home.so                                   {include if "with-homed"}
 password    requisite                                    pam_pwquality.so local_users_only
 password    sufficient                                   pam_unix.so yescrypt shadow {if not "without-nullok":nullok} use_authtok
 password    sufficient                                   pam_sss.so use_authtok
 password    required                                     pam_deny.so
+-password    sufficient                                  pam_systemd_home.so                                   {include if "with-homed"}
 
 session     optional                                     pam_keyinit.so revoke
 session     required                                     pam_limits.so
 session     optional                                     pam_ecryptfs.so unwrap                                {include if "with-ecryptfs"}
+-session    optional                                     pam_systemd_home.so                                    {include if "with-homed"}
 -session    optional                                     pam_systemd.so
 session     optional                                     pam_oddjob_mkhomedir.so                               {include if "with-mkhomedir"}
 session     [success=1 default=ignore]                   pam_succeed_if.so service in crond quiet use_uid
 session     required                                     pam_unix.so
 session     optional                                     pam_sss.so
 session     optional                                     pam_gnome_keyring.so only_if=login auto_start          {include if "with-pam-gnome-keyring"}
+-session    optional                                     pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
This ended up being duplicated now (also all other entries).

Reply:
Should be corrected now. Just have to squash again.